nss-day-cohort-48/rare-server-polite-pirates
# Jf add tag server

#### Changes Made

1. Modified file `posts/request.py` to include `tags to post` functionality. I modified the function get_all_posts to include the join table `PostTags`.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout jf-add-tag-server
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.

- [x] check that all modules mentioned above have been modified
- [x] run debugger
- [x] in postman get all posts
- [x] check and see if there is a "tags" key with an empty array "[]"
- [x] copy and paste this into your body of postman and do a POST method
 {
        "user_id": 2,
        "category_id": 2,
        "title": "Tag Test",
        "publication_date": 1627664528,
        "image_url": "a_picture",
        "content": "tags test",
        "approved": 0,
        "tags": [1, 2, 3]
        }

- [x] do a GET is postman and see if the 'tags' key has your tags in them! 

4. View code file.
   > Confirm file modifications are present as indicated above.
